core/src/tpu.rs

```diff
@@ -229,10 +231,18 @@ impl Tpu {
             prioritization_fee_cache,
         );
 
+        let (tpu_entry_sender, tpu_entry_receiver) = unbounded();
+        let tpu_entry_notifier = TpuEntryNotifier::new(
+            entry_receiver,
+            entry_notification_sender,
+            tpu_entry_sender,
+            exit,
+        );
```

Comment:
Would be nice to have this service only run if we need to track indexes, otherwise can avoid spinning up another thread